plugin-tester-java/build.gradle

```diff
@@ -1,49 +1,26 @@
+buildscript {
+  repositories {
+    mavenLocal()
+  }
+
+  dependencies {
+    classpath 'com.lightbend.akka.grpc:gradle-plugin:0.0.1'
```

Comment:
Should we call it akka-grpc-gradle-plugin?

Reply:
Yea. I think that way it will align with other plugins better.